_posts/blog/2016-08-21-gopherjs-1.7-1-release.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title:  "GopherJS 1.7-1 is released"
+date:   2016-08-21
+author: Dmitri Shuralyov
+---
+
+Hello gophers,
+
+Today, we are happy to make two announcements for GopherJS. First, GopherJS now [officially supports Go 1.7](https://github.com/gopherjs/gopherjs/issues/475), which is the current stable release of Go that came out recently. Second, GopherJS has its first named version attached to it, and we're calling it GopherJS 1.7-1. This makes it possible to have a release history tracking major changes and additions between versions.
```

Comment:
Maybe, maybe not. It's shows that the canonical issue tracking GopherJS' support for Go 1.7 is now resolved.

Would it be better if I updated it to point to that "event"? I.e.:

https://github.com/gopherjs/gopherjs/issues/475#event-762320892

![image](https://cloud.githubusercontent.com/assets/1924134/17840038/2388138c-67b0-11e6-8fa7-4a0ee30c0079.png)

Or maybe remove it altogether? I agree it's not the best first link for people to click on.